<launch>
  <include file="$(find pr2_machine)/$(env ROBOT).machine" />
  <include file="$(find pr2_moveit_config)/launch/planning_context.launch" />

  <node pkg="pr2_controller_manager" type="unspawner" args="l_arm_force_controller"  name="l_arm_force_controller_stopper" output="screen"/>
  <node pkg="pr2_controller_manager" type="pr2_controller_manager" args="start l_arm_controller" name="l_arm_controller_starter" output="screen"/>

  <node pkg="pr2_controller_manager" type="unspawner" args="r_arm_force_controller" name="r_arm_force_controller_stopper" output="screen"/>
  <node pkg="pr2_controller_manager" type="pr2_controller_manager" args="start r_arm_controller" name="r_arm_controller_starter" output="screen"/>

  <node pkg="rostopic" type="rostopic" args="pub -1 /catching_controller/reset humanoid_catching/Reset {}" name="rostopic_reset" output="screen"/>

  <node pkg="rostopic" type="rostopic" name="move_left_arm" args="pub l_arm_controller/joint_trajectory_action/goal pr2_controllers_msgs/JointTrajectoryActionGoal '{goal:{trajectory:{joint_names: [l_shoulder_pan_joint, l_shoulder_lift_joint, l_upper_arm_roll_joint, l_elbow_flex_joint, l_forearm_roll_joint, l_wrist_flex_joint, l_wrist_roll_joint], points: [{positions: [0.0, 0.0103108212, 0.0, -0.2939007067, 0.0, -0.4135609944, 0.0], velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], accelerations: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0] }]}}}' -1" />
  <node pkg="rostopic" type="rostopic" name="move_right_arm" args="pub r_arm_controller/joint_trajectory_action/goal pr2_controllers_msgs/JointTrajectoryActionGoal '{goal:{trajectory:{joint_names: [r_shoulder_pan_joint, r_shoulder_lift_joint, r_upper_arm_roll_joint, r_elbow_flex_joint, r_forearm_roll_joint, r_wrist_flex_joint, r_wrist_roll_joint], points: [{positions: [0.0, 0.0103108212, 0.0, -0.2939007067, 0.0, -0.4135609944, 0.0], velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], accelerations: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0] }]}}}' -1" />
</launch>
